unumpy/tests/test_numpy.py

```diff
@@ -195,6 +195,9 @@ def test_multiple_output(backend, method, args, kwargs):
     assert all(isinstance(arr, types) for arr in ret)
 
 
+dtypes = [onp.int8, onp.int16, onp.int32, onp.float32, onp.float64]
```

Comment:
Replace these with strings, e.g.
```suggestion
dtypes = ["int8", "int16", "int32", "float32", "float64"]
```